reconcile/saas_file_owners.py

```diff
@@ -334,6 +340,11 @@ def run(
         changed_path_matches = [
             c for c in changed_paths_copy if c.endswith(saas_file_path)
         ]
+        saas_file_target_path = diff["target_path"]
+        if saas_file_target_path:
+            changed_path_matches = [
```

Comment:
Perhaps I am not understanding this fully, Is overwriting   `changed_path_matches` really intended? What if I change both?

Reply:
great catch! https://github.com/app-sre/qontract-reconcile/pull/2595/commits/ae50a1633a2788faeb119224afb2be5815f7776a